website/docs/r/spring_cloud_new_relic_application_performance_monitoring.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+subcategory: "Spring Cloud"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_new_relic_application_performance_monitoring"
+description: |-
+  Manages a Spring Cloud Application Performance Monitoring resource for New Relic.
+---
+
+# azurerm_spring_cloud_new_relic_application_performance_monitoring
+
+-> **NOTE:** This resource is only applicable for Spring Cloud Service enterprise tier
+
+Manages a Spring Cloud Application Performance Monitoring resource for New Relic.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+resource "azurerm_spring_cloud_service" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "E0"
+}
+
+resource "azurerm_spring_cloud_new_relic_application_performance_monitoring" "example" {
+  name                            = "example"
+  spring_cloud_service_id         = azurerm_spring_cloud_service.example.id
+  app_name                        = "example-app-name"
+  license_key                     = "example-license-key"
+  agent_enabled                   = true
+  app_server_port                 = 8080
+  audit_mode_enabled              = true
+  auto_app_naming_enabled         = true
+  auto_transaction_naming_enabled = true
+  custom_tracing_enabled          = true
+  labels                          = "tagName1:tagValue1;tagName2:tagValue2;tagName3:tagValue3"
+  globally_enabled                = true
```

Comment:
These boolean properties all have default values, so can we omit them in the example to make it simple?